### ISSUE TITLE
Please use CCI (Condensed Creative Items) Library

### ISSUE THREAD

**Issue by ardissaps** [ConsistencyPlus/ConsistencyPlus#117]
It can be convenient because we have as said over 2000 item can be grouped with (+) symbol.
https://www.curseforge.com/minecraft/mc-mods/cci

**Comment by Siuolthepic** [comment]
I don't think this is going to be our solution to dealing with the issue. It doesn't support Forge at the moment and I'm more interested in using the method owolib uses. However, if Carter or other devs says otherwise I think it can be looked into

**Comment by ardissaps** [comment]
> I don't think this is going to be our solution to dealing with the issue. It doesn't support Forge at the moment and I'm more interested in using the method owolib uses. However, if Carter or other devs says otherwise I think it can be looked into

Owo is also good, it have button kategory outside of the creative inventory right? Like myticmetal?
